### ISSUE TITLE
Problem with multiple return values and assignments

### ISSUE THREAD

**Issue by jugglerchris** [kyren/luster#18]
I've noticed that assignments with functions with multiple return values doesn't work.  For example:

```lua
function mulret()
    return 1,2,3
end

a,b,c = mulret()
print(a,b,c)
```

In lua gives:
```
1       2       3
```

In Luster:
```
1       nil     nil
```

I think the problem is in the handling of assigment statements in the compiler, which seems not to have allowed for multiple returns yet (the function is called with 1 expected return value).

**Comment by kyren** [comment]
Oh, it might only work with local statements, not with assignment.  I'll look into this, thank you!

**Comment by kyren** [comment]
I think I just somehow forgot to do this when implementing the compiler, should be fixed now by 8549771

**Comment by jugglerchris** [comment]
Great, thanks! That was quick.
